src/NetscriptFunctions.ts

```diff
@@ -1836,6 +1838,17 @@ export const ns: InternalAPI<NSFull> = {
   printRaw: (ctx) => (value) => {
     ctx.workerScript.print(wrapUserNode(value));
   },
+  importFrom: (ctx) => (value) => {
+    const path = helpers.scriptPath(ctx, "path", value);
+    const server = ctx.workerScript.getServer();
```

Comment:
Getserver return should be checked, in case of things like a deleted server while running. Check other functions for an example.